.github/workflows/pytest.yaml

```diff
@@ -7,29 +7,28 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.7, 3.8, 3.9]
 
     steps:
       - uses: actions/checkout@v2
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+      - name: Install Poetry
+        uses: snok/install-poetry@v1
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pipenv flake8 wheel
```

Comment:
Just curious, as to why we want to do it like this? By installing specific dependencies? Would something like this be better?

```
      #----------------------------------------------
      # install dependencies if cache does not exist 
      #----------------------------------------------
      - name: Install dependencies
        if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
        run: poetry install --no-interaction --no-root
```

Reply:
Good catch! I forgot to change this after changing Pipenv. I've switched over to your suggestion in 41522b9.